### PR TITLE
Abs method 

### PR DESCRIPTION
**Pull request by sylwiabr**
This adds `abs` method for a LQuery value

### CODE REVIEW

**Comment by mwu-tow** [comment]
The code itself looks good. 
It'd be great if you added some tests. See https://github.com/luna/Dataframes/blob/master/native_libs/test/Tests.cpp#L381 for example how it was done with negate. The fixture deifiniton is here: https://github.com/luna/Dataframes/blob/master/native_libs/test/Tests.cpp#L279

**Comment by mwu-tow** [comment]
Also, likely a separate matter, but Luna's built-in types (Int, Real) should get this method as well. The assumption is that LQuery tries to mimic the built-in types interface.

**Comment by piotrMocz** [comment]
If you're doing a PR, please connect it to an issue (even if it meant creating one just for the sake of being connected to). ZenHub doesn't handle standalone PRs well ;)

File: native_libs/src/LQuery/AST.h [hunk 1]
```diff
@@ -45,7 +45,7 @@ namespace ast
     enum class ValueOperator
     {
         Plus, Minus, Times, Divide, Modulo,
-        Negate,
+        Negate, Abs,
```
Comment (pmlodawski) [inline native_libs/src/LQuery/AST.h]: Why not alphabetic order?

**Review by mwu-tow — APPROVED** [review]

**Pull request merged**